MiguelAValdesG/Hackerrank/JavaHashSet.java

```diff
@@ -0,0 +1,30 @@
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Scanner;
+
+public class JavaHashSet {
+
+  public static void main(String[] args) {
+    Scanner readLineTerminal = new Scanner(System.in);
+    HashSet<String> stringPairs = new HashSet<>();
```

Comment:
Always using interface type when declaring a collection
https://www.codejava.net/java-core/collections/18-java-collections-and-generics-best-practices